cmd/gogio/androidbuild.go

```diff
@@ -867,6 +874,17 @@ func getPermissions(ps []string) ([]string, []string) {
 	return permissions, features
 }
 
+func hasForegroundPermission(permissions []string) bool {
+	if fgPerms, ok := AndroidPermissions["foreground"]; ok {
+		for _, p := range permissions {
+			if p == fgPerms[0] { // []string, len 1
```

Comment:
Isn't fgPerms[0] a constant string? If so, use a Go constant here and for AndroidPermissions below.